ToDoList-frontend/src/app/bucket/bucket.component.ts

```diff
@@ -60,6 +67,25 @@ export class BucketComponent implements OnInit {
     this.fetchBucketTasks();
     this.fetchBucketTasksStates();
     this.fetchBucketTaskPriorities();
+    this.fetchAssigness();
+  }
+
+  showAssigneeList() {
+    this.showAssigneeListItems = !this.showAssigneeListItems;
+  }
+
+  selectAssignee(assignee: Assignee) {
+    setTimeout(() => {
+      if (this.showNewBucketTaskForm)
+      {
+        this.addNewBucketTaskFormGroup.patchValue({ assigneeControl:assignee.name });
```

Comment:
Jesli formularz dodawania i edycji taska wyglada podobnie, to prosciej by bylo zrobic jedna form grupe: `bucketTaskFormGroup` zamiast 2 osobnych.

Reply:
Świetny pomysł, zastosowałem to i kod się znacznie uprościł. Teraz mam pytanie - czy warto zrobić podobnie z modalami w html? Mam zagwozdke bo modal do dodawania i edycji różnią się nazwami ngForm. Co sądzisz?

Reply:
Warto, jeszcze jak ;)